src/main/java/baritone/pathing/movement/movements/MovementParkour.java

```diff
@@ -231,7 +257,7 @@ public MovementState updateState(MovementState state) {
                     // go in the opposite order to check DOWN before all horizontals -- down is preferable because you don't have to look to the side while in midair, which could mess up the trajectory
                     state.setInput(Input.CLICK_RIGHT, true);
                 }
-                if (dist == 3) { // this is a 2 block gap, dest = src + direction * 3
+                if (dist == 3 && !ascend) { // this is a 2 block gap, dest = src + direction * 3
```

Comment:
Why?

Reply:
This is akin to a sprintable ascend. When you are doing a non ascend distance 3, you're not sprinting and this code is so that you don't jump too late. While sprinting, making the jump is not an issue so we should jump earlier to land the jump without hitting the edge of the block and losing our forward velocity

Reply:
I see

Reply:
Can you add a comment explaining this?